scripts/assemble_changelog.py

```diff
@@ -219,12 +219,20 @@ def add_categories_from_text(self, filename, line_offset,
                                        category.name.decode('utf8'))
 
             body_split = category.body.splitlines()
+            _only_url_re = re.compile(br'^\s*\w+://\S+\s*$')
+            _has_url_re = re.compile(br'.*://.*')
             for line_number, line in enumerate(body_split, 1):
-                if len(line) > MAX_LINE_LENGTH:
+                if not _only_url_re.match(line) and \
+                   len(line) > MAX_LINE_LENGTH:
+                    long_url_msg = '. URL exceeding length limit must be ' \
+                        'alone in it\'s line.' if _has_url_re.match(line)  \
```

Comment:
Spelling: “**its** line” (“it's” is only an abbreviation of “it is” or “it has”)